### PR TITLE
Fix default layer host

### PR DESCRIPTION
**Pull request by PorterNan**
# What

Related to issue: https://github.com/Azure/communication-ui-library/issues/775
### Add a default host layer in base provider

1. make our lib compatible with react-full-screen
2. avoid polluting global dom tree
# Why

# How Tested

# Process & policy checklist

- [ ] I have updated the project documentation to reflect my changes if necessary.
- [ ] I have read the [CONTRIBUTING](https://github.com/Azure/communication-ui-library/blob/main/CONTRIBUTING.md) documentation.

**Is this a breaking change?**

- [ ] This change causes current functionality to break.

### CODE REVIEW

File: packages/react-composites/src/composites/common/BaseComposite.tsx [hunk 1]
```diff
@@ -80,10 +84,13 @@ export const BaseProvider = (
...
   registerIcons({ icons: { ...DEFAULT_COMPOSITE_ICONS, ...props.icons } });
 
+  // we use Customizer to override default LayerHost injected to <body />
+  // which stop polluting global dom tree and increase compatibility with react-full-screen
   const CompositeElement = (
     <FluentThemeProvider fluentTheme={fluentTheme} rtl={rtl}>
       <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, user-scalable=0" />
-      {props.children}
+      <Customizer scopedSettings={{ Layer: { hostId: globalLayerHostId } }}>{props.children}</Customizer>
+      <LayerHost id={globalLayerHostId} className={mergeStyles(globalLayerHostStyle)} />
```
[the remark this answers is not in the document]
Comment (JamesBurnside) [reply packages/react-composites/src/composites/common/BaseComposite.tsx:93]: This is amazing if so 😍

[the remark this answers is not in the document]
Comment (JamesBurnside) [reply packages/react-composites/src/composites/common/BaseComposite.tsx:93]: I assume when we assign layerHosts manually (such as for modals) that need to be in a specific layer they are unnaffected and not overriden by this?

**Review by JamesBurnside — APPROVED** [review]
❤️❤️❤️

File: packages/react-composites/src/composites/common/BaseComposite.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (PorterNan) [reply packages/react-composites/src/composites/common/BaseComposite.tsx:93]: > I assume when we assign layerHosts manually (such as for modals) that need to be in a specific layer they are unnaffected and not overriden by this?

Yes, according to my tests, it behaves just like in <body> tag, except we moving it into another div element, and all override is still working pretty well

**Pull request merged**